src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+#spring.datasource.url= jdbc:postgresql://localhost:5432/springbootdb
+#spring.datasource.username=postgres
+#spring.datasource.password=postgres@123
+#
+#spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
I realize this is commented out just to give developers an idea of how to make the db work without always having to make sure the schema objects have been set up properly in advance, but in the future we should consider having Liquibase hit the db on app startup and apply the necessary migrations. This will provide developers what they need to just spin up an empty postgres instance and expect the app to "just work", and will also allow CI/CD to "just work" in production.